### PR TITLE
Add support for php8

### LINKED ISSUE #251: PHP8 support
While updating a project with this package installed:

> Your requirements could not be resolved to an installable set of packages.
>
>  Problem 1
>    - ekino/newrelic-bundle[2.2.0, ..., 2.2.1] require php ^7.1 -> your php version (8.0.0) does not satisfy that requirement.
>    - Root composer.json requires ekino/newrelic-bundle ^2.2 -> satisfiable by ekino/newrelic-bundle[2.2.0, 2.2.1].

### PR DESCRIPTION
**Pull request by jderusse**
:crossed_fingers: fixes #251

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+#      - name: Upload coverage
+#        if: matrix.coverage == 'xdebug'
+#        run: |
+#          wget https://scrutinizer-ci.com/ocular.phar
+#          php ocular.phar code-coverage:upload --format=php-clover build/coverage.xml
+
+      - name: Stats
+        run: |
+          wget http://tnyholm.se/reporter.phar
+          php reporter.phar build:upload
```
Comment (Nyholm) [inline .github/workflows/ci.yml:98]: This can be removed

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+        env:
+          SYMFONY_PHPUNIT_REMOVE_RETURN_TYPEHINT: 1
+        run: |
+          ./vendor/bin/simple-phpunit ${{ matrix.phpunit }}
+
+#      - name: Upload coverage
+#        if: matrix.coverage == 'xdebug'
+#        run: |
+#          wget https://scrutinizer-ci.com/ocular.phar
+#          php ocular.phar code-coverage:upload --format=php-clover build/coverage.xml
```
Comment (Nyholm) [inline .github/workflows/ci.yml:93]: We can remove this

File: composer.json [hunk 3]
```diff
@@ -22,8 +22,8 @@
     },
     "require-dev": {
         "matthiasnoback/symfony-dependency-injection-test": "^3.1|^4.0",
-        "nyholm/symfony-bundle-test": "^1.6",
-        "symfony/phpunit-bridge": "^5.0",
+        "nyholm/symfony-bundle-test": "dev-master",
```
Comment (Nyholm) [inline composer.json:25]: This can be reverted. I'll release a new version now.

**Review by Nyholm — APPROVED** [review]
Awesome. Thank you 

Just remove a few lines and Im happy

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+          # dev
+          - php: 8.0
+            stability: 'dev'
+
+    steps:
+      - name: Set up PHP
+        uses: shivammathur/setup-php@2.7.0
+        with:
+          php-version: ${{ matrix.php }}
+          coverage: ${{ matrix.coverage }}
```
Comment (Nyholm) [inline .github/workflows/ci.yml:55]: Add this and we dont need to install symfony/flex ourselves. 

```suggestion
          coverage: ${{ matrix.coverage }}
          tools: flex
```

File: .github/workflows/ci.yml (hunk 1 above)
Comment (jderusse) [reply .github/workflows/ci.yml]: thanks

File: composer.json (hunk 3 above)
Comment (jderusse) [reply composer.json]: cool

**Pull request merged**